stdlib/base.pkl

```diff
@@ -1835,6 +1835,14 @@ class Listing<out Element> extends Object {
   @Since { version = "0.27.0" }
   external function getOrNull(index: Int): Element?
 
+  /// Returns the element at [index].
+  ///
+  /// Returns [default] applied to [index] if [index] is outside the bounds of this listing.
+  /// 
+  /// This is equivalent to `getOrNull([index]) ?? default.apply([index])`.
```

Comment:
The `[index]` notation creates a member link; it creates a code block and also a link. In a code snippet, we should just reference the parameter plainly.

```suggestion
  /// This is equivalent to `getOrNull(index) ?? default.apply(index)`.
```